Integrations/integration-iDefense.yml

```diff
@@ -0,0 +1,411 @@
+description: Accenture Security
+configuration:
+- display: URL
+  name: url
+  defaultvalue: https://api.intelgraph.verisign.com/
+  type: 0
+  required: true
+- display: API Token
+  name: api_token
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Do not validate server certificate (insecure)
```

Comment:
proxy?

Reply:
what do you mean?

Reply:
we have a flag to use system demisto proxy ettings or general one. you can see it at almost all of other integrations